e2e/playwright/tests/todo_plugin.spec.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// ***************************************************************
+
+import {expect, test} from '@e2e-support/test_fixture';
+import SlashCommandSuggestions from 'support/components/slash_commands';
+import {getTodoBotDMPageURL} from 'support/utils';
+
+export default {
+    connected: () => {
+        test.describe('available commands', () => {
+            test('with just the main command', async ({pages, page, pw}) => {
+               
+            const {adminClient, adminUser} = await pw.getAdminClient();
+            if (adminUser === null) {
+                throw new Error('can not get adminUser');
+            }
+            const dmURL = await getTodoBotDMPageURL(adminClient, '', adminUser.id);
+            await page.goto(dmURL, {waitUntil: 'load'});
+
+            const c = new pages.ChannelsPage(page);
+            const slash = new SlashCommandSuggestions(page.locator('#suggestionList'));
+
+            // # Run incomplete command to trigger help
+            page.getByTestId('post_textbox').pressSequentially('/todo');
```

Comment:
Note: This line was being performed using the postMessage() function exported from `e2e/playwright/support/utils.ts` but for some reason it fails with the error "TypeError: c.postMessage is not a function". This happens even in the `mattermost-plugin-github` repository. I'm not sure if some latest change in the `mattermost` repository is causing this to break.

Hence, I've used Playwright's `pressSequentially()` method to get past this issue. Let me know if you have any concerns/thoughts :) @mickmister @larkox

Reply:
This is resolved now.
It appears that the `postMessage()` method was removed from the `ChannelsPage` class during this commit https://github.com/mattermost/mattermost/commit/6109330bebd0f4a9022e28c047955a35fd56c0be#diff-d6fae6f34034f33e5d6182d647015326cd84b455120a953add37db8eab62fa3d which causes our method invocation to break. 
After discussions with @mickmister, we've decided to move ahead with using Playwright's `pressSequentially()` method to input text into message input box instead of relying on the methods from Mattermost/webapp repo.